src/AppBundle/Security/Firewall/TokenBearerListener.php

```diff
@@ -25,15 +25,14 @@ public function __construct(
         AuthenticationManagerInterface $authenticationManager,
         JWTTokenAuthenticator $jwtTokenAuthenticator,
         HttpMessageFactoryInterface $httpMessageFactory
-    )
-    {
+    ) {
         $this->tokenStorage = $tokenStorage;
         $this->authenticationManager = $authenticationManager;
         $this->jwtTokenAuthenticator = $jwtTokenAuthenticator;
         $this->httpMessageFactory = $httpMessageFactory;
     }
 
-    public function handle(GetResponseEvent $event)
```

Comment:
Unfortunately, `ListenerInterface` is [still declared](https://github.com/symfony/symfony/blob/b1ea2cd7a6b7f644a528f78f7814e01a7fabe181/src/Symfony/Component/Security/Http/Firewall/ListenerInterface.php) with `GetResponseEvent` 😢 

```
PHP Fatal error:  Declaration of AppBundle\Security\Firewall\TokenBearerListener::handle(Symfony\Component\HttpKernel\Event\RequestEvent $event) must be compatible with Symfony\Component\Security\Http\Firewall\ListenerInterface::handle(Symfony\Component\HttpKernel\Event\GetResponseEvent $event)
```

This interface is [deprecated since Symfony 4.3](https://github.com/symfony/symfony/blob/b1ea2cd7a6b7f644a528f78f7814e01a7fabe181/src/Symfony/Component/Security/Http/Firewall/ListenerInterface.php), they say "turn listeners into callables instead".
So you can try to update the usage, but it's totally fine if you just revert your changes 😉

Reply:
OK in fact. I revert changes for the `TokenBearerListener.php`. :)